src/components/cart/cart-actions/CartActions.tsx

```diff
@@ -0,0 +1,19 @@
+import './CartActions.scss';
+
+import React from 'react';
+
+type Props = {
+  clearCart: () => void;
+};
+
+const CartActions: React.FC<Props> = ({ clearCart }) => {
+  return (
+    <div className="cart-actions">
+      <button className="btn-clear" onClick={clearCart}>
+        Clear Cart
+      </button>
+    </div>
+  );
+};
+
+export default CartActions;
```

Comment:
**suggestion:** note you can just export directly your const. Named export is more compatible with tree-shaking.
Link: https://blog.neufund.org/why-we-have-banned-default-exports-and-you-should-do-the-same-d51fdc2cf2ad

Reply:
Insanely helpful, I did not pay that much attention to the way I did exporting before. I'm keeping this in mind!